prisma/schema.prisma

```diff
@@ -159,25 +159,30 @@ model Image {
 // this is for validator service
 model CandidUserInfo {
   id               Int     @id @default(autoincrement())
+  createdAt       DateTime @default(now())
+  updatedAt       DateTime @updatedAt
+
   claimId          Int?    @unique
   firstName        String?
   lastName         String?
   candid_entity_id String
   email            String
   profileURL       String
+  response         ResponseStatus?
 }
 
 model ValidationRequest {
-  id        Int      @id @default(autoincrement())
-  createdAt DateTime @default(now())
+  id                Int              @id @default(autoincrement())
+  createdAt         DateTime         @default(now())
+  updatedAt         DateTime         @updatedAt
 
   context           String
   validatorName     String
   validatorEmail    String
-  claimId           Int              // the id of the candid user claim (the root claim)
-  validationClaimId Int?             @unique // the id of validation claim
+  claimId           Int
+  validationClaimId Int?             @unique
   validationStatus  ValidationStatus @default(PENDING)
-  response          ResponseStatus?
+  rating            Int?
```

Comment:
rating woudl go with the user, or with the user-profile connection?  it coudl also be with each request i guess how valid is taht request

Reply:
rating is for validation and we use it to get the response color